x/ccv/provider/keeper/grpc_query_test.go

```diff
@@ -349,21 +359,29 @@ func TestGetConsumerChain(t *testing.T) {
 			strDenylist[j] = addr.String()
 		}
 
+		metadataLists[i] = types.ConsumerMetadata{Name: chainID}
+		pk.SetConsumerMetadata(ctx, consumerID, metadataLists[i])
+
+		phase := uint32(i)
```

Comment:
Should this be `mod (number of phases)` or something similar?

Reply:
Not needed since we have 5 phases for 4 chains iterated chains.